integration-tests/src/main/resources/data/NorthwindDataAdapter.jrdax

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Should this file reside with the .jrxml files?  I was thinking the styles would also live there (that's how my projects are structured, all the Jasper-related stuff in it's own folder).

The data files living in their own folder is a great idea.

Reply:
I am using their example exactly as it is in their `/samples` so I am leaving it where it was in /data ?

Reply:
:shrug: